docs/reference/catalog-reference.md

```diff
@@ -1,6 +1,360 @@
 # Catalog
 
-::: up42.catalog.Catalog
-    selection:
-        inherited_members: True
-        members: {{ funcs_catalog }}
+The Catalog class enables access to the UP42 [catalog functionality](../../catalog/).
```

Comment:
nitpick:
```suggestion
The Catalog class enables access to the UP42's [catalog functionality](../../catalog/).
```

Reply:
Hey @seedlit as per our [writing style](https://up42.atlassian.net/wiki/spaces/DT/pages/3453354036/Punctuation#Apostrophe) we don't use possessive apostrophes when referring to an UP42-specific functionality/element.